src/util/string_utils.cpp

```diff
@@ -38,7 +38,7 @@ std::string strip_string(const std::string &s)
     return "";
 
   std::string::const_reverse_iterator r_it;
-  for(r_it=s.rbegin(); std::isspace(*r_it); r_it++);
+  for(r_it=s.rbegin(); std::isspace(*r_it); r_it++) {}
```

Comment:
I'm starting to wonder whether you might want to use `first_not_of("\n\t ")` and `last_not_of("\n\t ")` ?